lib/runners/recommended_config.rb

```diff
@@ -0,0 +1,26 @@
+module Runners
+  module RecommendedConfig
+    def warn_recommended_config_file_release(config_filename, release_date)
+      add_warning <<~MSG, file: config_filename
+        Sider's recommended configuration file is about to be released in #{release_date}.
```

Comment:
Sorry, I should have noticed it sooner. According to the name of the arg `release_date`, I think "on" should be used instead of "in."

```suggestion
        Sider's recommended configuration file is about to be released on #{release_date}.
```

Reply:
I choose an ambiguous target date as "in mid Oct 2020" because we notice about 2 weeks prior to the release.
In case of strict date of the release, e.g. "on 12 October", I will move the preposition word into the variable `release_date`.

Reply:
That's understandable. `*_date` in `release_date` confused me. Perhaps, the variable should be renamed to another, but I'm not sure.

Reply:
How about `deadline`? I found the word in `processor.rb`.

Reply:
It looks great :+1: